zerver/data_import/import_util.py

```diff
@@ -57,6 +57,37 @@ def build_recipient(type_id: int, recipient_id: int, type: int) -> ZerverFieldsT
     recipient_dict = model_to_dict(recipient)
     return recipient_dict
 
+def build_recipients(zerver_userprofile: List[ZerverFieldsT],
+                     zerver_stream: List[ZerverFieldsT]) -> List[ZerverFieldsT]:
+    recipient_id = 1
+    recipients = []
+
+    for user in zerver_userprofile:
+        type_id = user['id']
+        type = Recipient.PERSONAL
+        recipient = Recipient(
+            type_id=type_id,
+            id=recipient_id,
+            type=type,
+        )
+        recipient_dict = model_to_dict(recipient)
+        recipients.append(recipient_dict)
+        recipient_id += 1
+
+    for stream in zerver_stream:
+        type_id = stream['id']
+        type = Recipient.STREAM
+        recipient = Recipient(
+            type_id=type_id,
+            id=recipient_id,
+            type=type,
+        )
+        recipient_dict = model_to_dict(recipient)
+        recipients.append(recipient_dict)
+        recipient_id += 1
+
+    return recipients
```

Comment:
I assume the longer-term plan is to reuse this function for the Slack/Gitter imports and cut a bunch of code duplication?  Seems reasonable.

Reply:
yeah, I added a relevant comment